WordPress/gutenberg
# Autocomplete: Announce results to screen readers when first becoming visible

## What?


Changes the `speak()` function to fire when autocomplete items change. This allows screen reader users to know how many results exist initially before filtering.

## Why?


Fixes #5592

## How?


Modify the code and add an E2E test.

## Testing Instructions





1. Open a post or page.
2. Add a new block.
3. Type "/".
4. Notice how the `aria-live="assertive"` region at the bottom of the page announces the initial 9 results. This uses debounce, wait a few seconds.
5. Enter another keyword and ensure the results are updated.
6. Pressing escape should close suggestions list.

### Testing Instructions for Keyboard


Same.

## Screenshots or screencast
